src/index.html

```diff
@@ -11,12 +11,87 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a
+          href="#"
+          class="img"
+        >
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <a
+          class="nav__link is-active"
+          href="#"
+        >
+          Apple
+        </a>
+        <a
+          class="nav__link"
+          href="#"
+        >
+          Samsung
+        </a>
+        <a
+          class="nav__link"
+          href="#"
+        >
+          Smartphones
+        </a>
+        <a
+          class="nav__link l"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'l' does not communicate the purpose or meaning of the element it's applied to.